www/include/monitoring/status/Hosts/host.php

```diff
@@ -187,7 +187,30 @@
 );
 
 $deprecationMessage = _('[Page deprecated] Please use the new page: ');
-$redirectionUrl = $resourceController->buildListingUri([]);
+$jsonFilter = '{
```

Comment:
to make it more readable, you should use `json_encode` on a php array
furthermore, following properties are not mandatory : 
* id
* name
* criterias.statuses
* criterias.hostGroups
* criterias.serviceGroups